layers/+readers/pdf/README.org

```diff
@@ -95,6 +113,7 @@ Due to =image-mode=’s behaviour, the =pdf-tools= buffer might go back to the
 first page after switching windows. If this happens, you can quickly go back to
 the previous page in history with =pdf-history-backward= (~``~ in Evil state and
 ~B~ in Emacs state.)
+On some machines, pdf-tools can be quite slow (especially with a line-number mode active). You might want to try alternatives such as zathura.
```

Comment:
A short link or explanation how to install `zathura` and why it is better/faster would be useful in the docs.

Reply:
Zathura is not an Emacs package though. Should I add some explanation/links?
Zathura is basically a specialized pdf viewer with a vim-like interface. It’s fast. And it works pretty much out of the box. It’s available on both macOS and linux, though I doubt Windows support.

Reply:
I think in this case it is enough to say to use an external PDF reader though it somehow defeats the purpose, I mean the idea is to do that with emacs otherwise I could always use the native PDF reader of the respective OS, I think all big three have one now a days.